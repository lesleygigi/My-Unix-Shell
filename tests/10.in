tail -v -n +1 tests/test_dir/file1 tests/test_dir/file2 tests/test_dir/file3 tests/test_dir/file4
