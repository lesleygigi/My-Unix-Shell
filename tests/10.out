tail -v -n +1 tests/test_dir/file1 tests/test_dir/file2 tests/test_dir/file3 tests/test_dir/file4
==> tests/test_dir/file1 <==
file1 content
more text

==> tests/test_dir/file2 <==
content of file2
content of file2
content of file2
another line
file2: more lines in file2
the last line of file2

==> tests/test_dir/file3 <==
This is file3

==> tests/test_dir/file4 <==
